RELEASES.md

```diff
@@ -1,18 +1,13 @@
-# Draft: Needs community review
-This document is currently a work in progress, and will need more feedback from the community before it can be finalized.
+The overarching goal of this release policy is to make sure that developers know what to expect when they use Arizona Quickstart for their websites. Arizona Quickstart has adopted semantic versioning to allow site maintainers to keep websites up to date with security updates and other critical improvements without having to track the ongoing feature development of Arizona Quickstart.
 
-The overarching goal of this release policy is to make sure that developers know what to expect when they use Quickstart 2.0 for their websites. Quickstart 2.0 has adopted semantic versioning to allow site maintainers to keep sites up to date with security updates and other critical improvements without having to track the ongoing feature development of Quickstart. 
-
-*This policy explains what changes can be made to Quickstart following the release of 2.0.0.*
 ## Release Overview
-Quickstart 2 introduces a six-month minor release cycle with releases targeted for March and September. Minor releases provide new improvements and functionality without breaking backward compatibility and will usually include the latest minor release version of Drupal.
+Arizona Digital will support two minor releases at a time with ongoing patch releases. This process will allow for flexibility of development while allowing site owners to have more stable websites with fewer ongoing updates.
 
-Arizona Digital will support two minor releases at a time with ongoing patch releases.
+Note: When an Arizona Quickstart minor release contains a major or minor Drupal core release, it will be *vital* that existing sites are updated to the latest patch release of the previous minor version of Arizona Quickstart and ensure that Drupal database updates have been applied BEFORE updating to the new minor release.
```

Comment:
Maybe this part could be italics?
```suggestion
_Note: When an Arizona Quickstart minor release contains a major or minor Drupal core release, it will be *vital* that existing sites are updated to the latest patch release of the previous minor version of Arizona Quickstart and ensure that Drupal database updates have been applied BEFORE updating to the new minor release._
```